src/interactive-window/interactiveWindowProvider.ts

```diff
@@ -124,15 +124,28 @@ export class InteractiveWindowProvider
                 return;
             }
 
+            const tab = interactiveWindowMapping.get(iw.uriString);
+            const notebookEditor = window.visibleNotebookEditors.find(
```

Comment:
`window.visibleNotebookEditors` is probably not safe when we have multiple IW windows.